src/App/Controls/AvatarImageSource.cs

```diff
@@ -85,6 +89,25 @@ private Stream Draw()
             return SKImage.FromBitmap(bitmap).Encode(SKEncodedImageFormat.Png, 100).AsStream();
         }
 
+        private string GetFirstLetters(string data, int count)
+        {
+            var parts  = data.Split();
+            if (parts.Length > 1)
+            {
+                var text = "";
+                for (int i = 0; i < count; i++)
+                {
+                    text += parts[i].Substring(0,1);
+                }
+                return text;
+            }
+            if (data.Length > 2)
+            {
+                return data.Substring(0, 2);
+            }
+            return null;
+        }
```

Comment:
Could we make `count` a little bit more specific? I think it's not directly understood of what that means from the caller. Because as far as I can see `count` is the number of first letters you want to return if the `data` can be splitted. 
Also this method can break if `count` > than the number of parts of `data`, e.g. `data.Split()` return 2 parts and you send 4 as count then the `parts[i].Substring(0,1)` will throw Index out of bounds exception.
Finally, I think the method should consider `count` as the final condition and Substring to comply with the signature and what that means (or at least what I think it means)
```c#
if (data.Length > count)
{
    return data.Substring(0, count);
}
```

Reply:
I didn't want to deviate too much from the jslib version but I updated the var name and added protection against going out of bounds.